src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -87,7 +87,6 @@ const createAuthenticatorOptions = (options = []) => {
       label: option.label,
       value: valueObject,
       relatesTo: option.relatesTo,
-      authenticatorType: option.relatesTo?.type,
```

Comment:
Doesnt hurt to leave it here.  @haishengwu-okta  what do you think

Reply:
Prefer to remove as long as it is not used at all.
We can add it back just in case we need it in future.